ChattoAdditions/Source/UI Components/CircleProgressIndicatorView/CircleIconView.swift

```diff
@@ -0,0 +1,271 @@
+//
+//  CircleIconView.swift
+//  ChattoAdditions
+//
+//  Created by Mikhail Gasanov on 29/05/2019.
+//  Copyright © 2019 Badoo. All rights reserved.
+//
+
+import UIKit
+
+public enum CircleIconType {
```

Comment:
It's a great opportunity to clean up this component and remove all unused code. We don't use the most of its functionality.

Reply:
After a discussion, we decided to leave everything as it is.